assets/js/modules/analytics-4/utils/validation.js

```diff
@@ -31,7 +31,9 @@ import { isValidNumericID } from '../../../util';
  * @return {boolean} Whether or not the given property ID is valid.
  */
 export function isValidPropertyID( propertyID ) {
-	return typeof propertyID === 'string' && /^\w+$/.test( propertyID );
+	return (
+		typeof propertyID === 'string' && /^[a-zA-Z0-9-]+$/.test( propertyID )
```

Comment:
For GA4, property IDs have always been numeric, no? Is this being used on a measurement ID somewhere perhaps?

Reply:
Ah.. yes, you are right. Updated.